code/game/machinery/stasis.dm

```diff
@@ -38,6 +40,14 @@
 	. += "<span class='notice'>Alt-click to [stasis_enabled ? "turn off" : "turn on"] the machine.</span>"
 	. += "<span class='notice'>[src] is [op_computer ? "linked" : "<b>NOT</b> linked"] to an operating computer.</span>"
 
+/obj/machinery/stasis/proc/initial_link()
+	for(var/direction in GLOB.alldirs)
```

Comment:
by the time this is invoked you may have already been linked.
Also applied to initial link bellow.